app/views/admin/user_groups/_form.haml

```diff
@@ -1,4 +1,15 @@
 .row
-  .col-8
+  .col-sm-6
     = f.input :name
+
     = f.input :user_ids, as: :select_two, label_method: :name, collection: User.active.not_system, label: 'Users', input_html: { multiple: true }
+  .col-sm-6
+    .well.mt-6
+      %p
+        It is often helpful to name this group for the agency where the people included in the group work and their role in that agency.  Some examples:
+        %strong "Blue Mountain Shelter: Providers", "Teal House: Report Runners", or "Administrators: MA-500".
+        Using the pattern
+        %strong "Agency Name: User Function"
+        will group user groups together by agency, inverting that to
+        %strong "Function: Agency"
+        will group user groups by function.
```

Comment:
I think this language is probably overdoing it.. to me it implies that some functional grouping actually happens based on the naming convention. Really it just a convenience for how things appear when sorting by name.

I think it might just be better to list out a couple examples, like:
```
* Blue Mountain Shelter: Providers
* Blue Mountain Shelter: Site Managers
```